packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -352,7 +357,17 @@ Future<XcodeBuildResult> buildXcodeProject({
   }
 }
 
-Future<Null> diagnoseXcodeBuildFailure(XcodeBuildResult result, BuildableIOSApp app) async {
+String readGeneratedXCConfig(String appPath) {
+  String generatedXCConfig;
+  final String flutterPodWrapperPath =
```

Comment:
Why is this called flutterPodWrapperPath? There's a bunch of general Flutter flags. Just call it generateXCConfigPath